ui/restHandler_test.go

```diff
@@ -0,0 +1,25 @@
+package ui // import "fortio.org/fortio/ui"
+
+import (
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+const emptyResult = `{"results":null}`
+
+func TestEmptyResult(t *testing.T) {
```

Comment:
it's going to be hard to do meaningful tests without actually starting some servers (no we won't start having mock objects for everything, let's not go down that path :) )

you can see e2e tests in fhttp/

ui/ has had no go test in part for the bad reason of not killing the coverage % (!) and in part because most of that is excercised somewhat in Webtest.sh

I think we can move the resHandler to fhttp if we're going to properly test it - which we should - eventually

Reply:
I'm doing that disentangling in https://github.com/fortio/fortio/pull/597 as it's really my mess to clean up

Reply:
#611 has tests ptal